DocSum/docsum.py

```diff
@@ -19,6 +23,16 @@ def __init__(self, host="0.0.0.0", port=8000):
         self.megaservice = ServiceOrchestrator()
 
     def add_remote_service(self):
+
+        data = MicroService(
```

Comment:
Can we make all multimedia related things optional?  This kind of last minute changes which introduces new required components will break the existing K8S(and helm chart) deployment, and caught us in surprise. We need time to add new components support in k8s(i.e. a2t, v2a, multimedia2text).